server/src/store.js

```diff
@@ -1,16 +1,16 @@
 import uuid from 'uuid';
-import mongoose from 'mongoose';
-import { SORT_PARAM_DATE, SORT_PARAM_SCORE } from 'Shared/query_params';
+import {connect, connection} from 'mongoose';
+import { SORT_PARAM_DATE, SORT_PARAM_SCORE } from './shared/query_params';
 import Bit from './Bit';
 import User from './User';
 
 const DEFAULT_PAGE_SIZE = 25;
 
-mongoose.Promise = global.Promise;
-mongoose.connect(process.env.STORE_URI);
+// mongoose.Promise = global.Promise;

```

Comment:
I think we want this so Mongoose uses real Promises: https://mongoosejs.com/docs/promises.html#plugging-in-your-own-promises-library

Reply:
I think since we use mongoose 5 we don't need this. https://stackoverflow.com/a/51862948/2875073